ResourceGenerator/settings.template.json

```diff
@@ -0,0 +1,6 @@
+{
+    "resourceName": "<ResourceFriendlyName>",
+    "description": "This resource configures an <ResourceDescription> Device.",
```

Comment:
I would remove "device" from the template. This might be confusing, if we generate other resources with the DRG.

Reply:
Will update in the evening